### PR TITLE
#723 Structure with some S-groups cannot be opened

### PR DESCRIPTION
**Pull request by AndreiProkofev**

### CODE REVIEW

File: packages/ketcher-react/src/script/ui/state/shared.js [hunk 1]
```diff
@@ -54,60 +54,64 @@ export function load(struct, options) {
...
-            if (stereoProp) {
-              atom.stereoLabel = stereoProp.stereoLabel
-              atom.stereoParity = stereoProp.stereoParity
-            }
+            editor.struct(struct)
+          }
+        },
+        err => {
+          //TODO: add error handler call
+          if (!err.isSGroupError) {
```
Comment (AndreiProkofev) [inline packages/ketcher-react/src/script/ui/state/shared.js:110]: The only change in this file. Other stuff is just code styling from prettier

**Review by AndreiMazol — CHANGES_REQUESTED** [review]
Confirm dialog should be shown once even if file contains a lots of unsupported S-groups

File: packages/ketcher-react/src/script/ui/state/shared.js (hunk 1 above)
Comment (AndreiProkofev) [reply packages/ketcher-react/src/script/ui/state/shared.js]: Deprecated

**Comment by AndreiProkofev** [comment]
PR closed due to migration of the solution to release 2.3.1

**Pull request closed without merging**